src/coreclr/tools/Common/Compiler/ObjectWriter/MachObjectWriter.cs

```diff
@@ -380,6 +402,24 @@ protected internal override unsafe void EmitRelocation(
             string symbolName,
             long addend)
         {
+            // We don't emit the range node name into the image as it overlaps with another symbol.
+            // For relocs to it, instead target the start symbol.
+            // For the "symbol size" reloc, we'll handle it later when we emit relocations in the Mach format.
```

Comment:
Can you add details to readytorun-platform-native-envelope.md about the issues/decisions like this related to the format? Symbol ranges, relocs, defining a base symbol, which sections different information is emitted into.